extensions_build_config.bzl

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/envoyproxy/envoy/blob/master/bazel/README.md#disabling-extensions for details on how this system works.
+EXTENSIONS = {
```

Comment:
Is there a complete list of extensions somewhere? I would like to take a look at what we are disabling and cross-check it with the functionality we are promising and using.

Reply:
The full set is contained over at https://github.com/envoyproxy/envoy/blob/master/source/extensions/extensions_build_config.bzl